src/palace/manager/api/sip/client.py

```diff
@@ -231,8 +231,6 @@ class Constants:
 class SIPClient(Constants, LoggerMixin):
     # Maximum retries of a SIP message before failing.
     MAXIMUM_RETRIES = 5
-    # Timeout in seconds
-    TIMEOUT = 3
```

Comment:
Minor suggestion: Instead of removing this, could we just rename to DEFAULT_TIMEOUT and use it below, instead of hardcoding `3` down there?
```python
    # Default timeout in seconds, used when no timeout explicitly configured.
    DEFAULT_TIMEOUT =  3
```